---


 src/dxvk/dxvk_options.h    |  6 ++++++
 8 files changed, 107 insertions(+), 13 deletions(-)

diff --git a/src/dxvk/dxvk_options.h b/src/dxvk/dxvk_options.h
index 5351ac68..5569a9b7 100644
--- a/src/dxvk/dxvk_options.h
+++ b/src/dxvk/dxvk_options.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "../util/config/config.h"
+#include "dxvk_include.h"
 
 namespace dxvk {
 
@@ -27,6 +28,11 @@ namespace dxvk {
     /// Enables pipeline lifetime tracking
     Tristate trackPipelineLifetime = Tristate::Auto;
 
+    // Enable async pipelines
+    bool enableAsync;
+    // Enable state cache with gpl and fixes for async
+    bool gplAsyncCache;
+
     /// Shader-related options
     Tristate useRawSsbo = Tristate::Auto;
 
-- 
2.48.1

